script.js

```diff
@@ -1418,6 +1419,14 @@ let projectData = [
      projectName: 'Captcha Validator',
      projectImage: 'assets/Images/CaptchaValidator.png',
      projectUrl: 'public/CaptchaValidator.html'
+   },
+   {
+    projectName: "Filterable Image Gallery",
+    projectImage: "assets/Images/Filterable_Image_Gallery/Filterable_Image_Gallery.png",
+    projectUrl: "public/Filterable_Image_Gallery.html",
+  }
+  
+]
```

Comment:
This is ending the list which is introducing error. Remove the square bracket.
But there are a lot of projects down below.